classes/collector/manager.php

```diff
@@ -49,9 +49,16 @@ public static function send_metrics(array $items) {
             try {
                 if ($collector->is_ready()) {
```

Comment:
We are missing logic here, if it is not enabled then it shouldn't be checked. If it was checked and was failing and you disable it then it is still failing

Reply:
I also think there might be a logic but with the original code, when sending metrics the collector must be both is_enabled and is_ready

Reply:
On line 49 there is this: 

     $plugins = cltr::get_enabled_plugins();

At the collector level we only have is_ready() available. Looks like I will need to duplicate a fair bit of this code on the check side of things so that if somebody uninstalls a collector we'll ignore the results for any of its children. Should I also delete them from the config?

Reply:
ok lets ignore this for now I might retest this when this has landed and make a followup issue